### PR TITLE
Add new version of VSCode Java - 0.57.0

### PR DESCRIPTION
**Pull request by svor**
Signed-off-by: svor <vsvydenk@redhat.com>

### What does this PR do?
Adds new version of VSCode Java - 0.57.0

### Related issue:
https://github.com/eclipse/che/issues/16051

### CODE REVIEW

**Comment by nickboldt** [comment]
I've merged this, but I think it's incomplete - doesn't solve two problems:

* 0.55.1 should be marked deprecated and force everyone to update to 0.57.0
* quarkus plugin relies on 0.55.1 so I'm assuming that should ALSO be updated w/ deprecation path to 0.57.0

**Comment by ericwill** [comment]
LGTM but Nick already merged. :+1:

**Comment by nickboldt** [comment]
Yes... But see follow-up.

Should we do that too, as a blocker for 7.9.0?

**Comment by ericwill** [comment]
> Yes... But see follow-up.
> 
> Should we do that too, as a blocker for 7.9.0?

Is it mandatory we deprecate 0.55.1 for 7.9.0? Or can we leave it for 7.10.0? IMO it's best to only change what's absolutely necessary, and do more thorough testing for Quarkus on 0.57.0 to make sure nothing breaks there.

File: v3/plugins/redhat/java11/0.57.0/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+apiVersion: v2
+publisher: redhat
+name: java
```
Comment (nickboldt) [inline v3/plugins/redhat/java11/0.57.0/meta.yaml]: should this be java11 ?

File: v3/plugins/redhat/java8/0.57.0/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,23 @@
+apiVersion: v2
+publisher: redhat
+name: java
```
Comment (nickboldt) [inline v3/plugins/redhat/java8/0.57.0/meta.yaml]: should this be java8 ?

**Comment by svor** [comment]
@nickboldt it should be 
https://github.com/eclipse/che-plugin-registry/pull/391

**Comment by nickboldt** [comment]
thanks, fixed in 7.9.x (and 7.9.0 rebuilt); merged in master too.

**Comment by nickboldt** [comment]
followup work in https://github.com/eclipse/che/issues/16107

**Pull request merged**